Cargo.toml

```diff
@@ -15,7 +15,7 @@ path = "src/main.rs"
 nix = "*"
 libc = "*"
 tempdir = "*"
-yaml-rust = "*"
+yaml-rust = { git = "https://github.com/hallettj/yaml-rust", branch = "load_from_str_with_markers" }
```

Comment:
I think we should pin this to a commit that is guaranteed to compile. (I think you can do that with a `rev` field.)

Reply:
Ok, I have made that change. The revision is locked in `Cargo.lock` - but I do see the benefit of giving a revision in `Cargo.toml` to make sure that nothing breaks when someone runs `cargo update`.